### PR TITLE
Print statistics how many files were checked in verbose

### LINKED ISSUE #8935: Print statistics how many files were checked in verbose.
### Current problem

I am trying to fine tune pylint and when dealing with excludes I see that sometimes too many files were excluded and would like to compare with number of files in a project.

### Desired solution

After pylint completed I would like to have printed something similar to black:
```
black .
2 files reformatted, 13493 files left unchanged.
```

```
pylint *.py **/*.py --verbose
...
13493 files checked.
```

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by meltemkenis**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✅ |  Other          |

## Description
While using verbose flag users want to see how many files are included or excluded.
With this PR, after score details, users will see how many files are included or excluded. Example:

`Your code has been rated at 7.5/10`
`Checked 1 files, skipped 0 files`

Closes #8935

### CODE REVIEW

**Comment by meltemkenis** [comment]
@DanielNoord @Pierre-Sassoulas hello, can I get some reviews/ feedbacks for this PR, please?

**Review by DanielNoord — DISMISSED** [review]
Sorry, we dropped the ball here.

Could you add a news fragment using `towncrier`?

**Review by DanielNoord — COMMENTED** [review]
Sorry, I had to be more specific. Please see the [docs](https://pylint.readthedocs.io/en/latest/development_guide/contributor_guide/contribute.html#creating-a-pull-request).

You'll need to use `towncrier create <IssueNumber>.<type>` to create a fragment specific to this PR.

File: doc/whatsnew/3/3.1/index.rst [hunk 1]
```diff
@@ -12,5 +12,25 @@
 Summary -- Release highlights
 =============================
 
-
 .. towncrier release notes start
+
+What's new in Pylint 3.1.0?
```
Comment (DanielNoord) [inline doc/whatsnew/3/3.1/index.rst]: Could you remove these changes?

File: doc/whatsnew/fragments/8935.other [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Print how many files were checked in verbose.
```
Comment (DanielNoord) [inline doc/whatsnew/fragments/8935.other:1]:
```suggestion
Print how many files were checked in verbose mode.
```

**Review by DanielNoord — COMMENTED** [review]
One last nit!

**Review by DanielNoord — DISMISSED** [review]
Thanks, very sensible change!

And thanks for the ping 😄

**Comment by meltemkenis** [comment]
thank you for your review! 🙂

**Comment by DanielNoord** [comment]
@meltemkenis 

Ah, you'll need to change the fragment slightly to get `pre-commit` to pass.

**Review by DanielNoord — APPROVED** [review]

**Pull request merged**